README.md

```diff
@@ -91,6 +91,29 @@ S, val, vec = parallel_structure_tensor_analysis(data, sigma, rho, devices=4*['c
 
 The ideal block size depends on the `sigma` and `rho`, the devices, and the memory available for the devices. Usually values between 100 and 400 work well. **If you encounter out-of-memory errors, try reducing the block size and/or the number of processes.**
 
+You can use the following snipped to get a progress bar (with [tqdm](https://github.com/tqdm/tqdm#hooks-and-callbacks))
+
+``` python
+class STETqdm(tqdm):
```

Comment:
How about calling the class `TqdmTotal` which aligns more with [their examples](https://github.com/tqdm/tqdm?tab=readme-ov-file#hooks-and-callbacks)?